app/views/packages/_form.html.erb

```diff
@@ -11,158 +11,219 @@
-    <h4>Create New Record or Merge URL Field</h4>
-    <%= f.radio_button :proc_type, "newonly" %>
-    <%= f.label :proc_type_newonly, "New only Always create new or overlay, no search for matching print record in Symphony" %>
-    <%= f.radio_button :proc_type, "mergenew" %>
-    <%= f.label :proc_type_mergenew, "Merge or New Create new only if no overlay match or merge match with field(s) below" %>
-    <%= f.radio_button :proc_type, "mergeonly" %>
-    <%= f.label :proc_type_mergeonly, "Merge only Never create new, merge URL if match found with field(s) below" %>
-    <%= f.radio_button :proc_type, "ckeymerge" %>
-    <%= f.label :proc_type_ckeymerge, "CKEY-URL Merge Merge URL from ckey-URL input list" %>
-  </div>
-  <div class="field">
-    <h4>Fields to Match Incoming to Symphony Records</h4>
-    <p><em>At least one required if "Merge or New" or "Merge only" selected above. Not used in "New only" or "CKEY-URL Merge"</em></p>
-    <%= f.check_box :match_opts, { :multiple => true }, '020', nil %>
-    <%= f.label :match_opts_020, '020 (subfield a,z) to Symphony 020 or 776' %>
-    <%= f.check_box :match_opts, { :multiple => true }, '776_isbn', nil %>
-    <%= f.label :match_opts_776_isbn, '776 (subfield z) to Symphony 020 or 776' %>
-    <%= f.check_box :match_opts, { :multiple => true }, '022', nil %>
-    <%= f.label :match_opts_022, '022 (subfield a,y,z) to Symphony 022 or 776' %>
-    <%= f.check_box :match_opts, { :multiple => true }, '776_issn', nil %>
-    <%= f.label :match_opts_776_issn, '776 (subfield x) to Symphony 022 or 776' %>
-    <%= f.check_box :match_opts, { :multiple => true }, '024_isbn', nil %>
-    <%= f.label :match_opts_024_isbn, '024 (subfield a,z) to Symphony 020 or 776' %>
+  <div class="tr">
+    <div class="td"><%= f.label 'Documentation URL' %></div>
+    <div class="td"><%= f.text_field :package_url, :size => 75 %></div>
   </div>
+</div>
 
-<h2>Processing Information</h2>
-  <div class="field">
-    <%= f.label 'Symphony URL Destination' %>
-    <%= f.select :url_field, ['856', '956'] %>
+<div class="table">
+  <div class="tr">
+    <div class="td"><h3>Pickup Information</h3></div>
   </div>
-  <div class="field">
-    <%= f.label 'Vendor ID Field (Read)' %>
-    <%= f.select :vendor_id_read, ['001', '090'] %>
+  <div class="tr">
+    <div class="td"><%= f.label :data_pickup_type %></div>
+    <div class="td"><%= f.select :data_pickup_type, ['FTP', 'AFS', 'FTP to AFS'] %></div>
   </div>
-  <div class="field">
-    <%= f.label 'Add Access Restriction Note' %>
-    <%= f.select :access_note, ['No', 'Yes'] %>
+  <div class="tr">
+    <div class="td"><h4>Option for "AFS" and "FTP to AFS" Data Pickup Types</h4></div>
   </div>
-  <div class="field">
-    <%= f.label 'Create/Update 040 with CSt' %>
-    <%= f.select :update_040, ['No', 'Yes'] %>
+  <div class="tr">
+    <div class="td"><%= f.label 'AFS Directory Path' %></div>
+    <div class="td"><%= f.text_field :afs_path, :size => 75 %></div>
```

Comment:
The new rails way is
```size: 75%```
instead of
```
size => 75%
```

but we don't have to change any of these...just an fyi.